src/components/posts/Facebook/socialMedia/socialMedia.css

```diff
@@ -116,5 +116,49 @@
   flex-direction: column;
   justify-content: space-evenly;
   align-items: center;
-  padding: 10px;
+  padding: 15px 15px 0 15px;
+}
+
+.ad-image {
```

Comment:
Can we make this responsive? Size gets wonky on smaller sizes
<img width="271" alt="Screen Shot 2021-07-21 at 10 16 23 PM" src="https://user-images.githubusercontent.com/65370631/126592807-0a3d213b-7def-4d35-b4a9-c445d1f024de.png">